utils/install_server.sh

```diff
@@ -73,6 +73,16 @@ if [ "$(id -u)" -ne 0 ] ; then
 	exit 1
 fi
 
+#bail if this system is managed by systemd
+_pid_1_exe="$(readlink -f /proc/1/exe)"
+if [ ${_pid_1_exe##*/} = systemd ]
```

Comment:
Quote `"${_pid_1_exe##*/}"` to prevent globbing and word splitting

Reply:
Could use $UID built-in readonly instead of $(id -u)

Reply:
@mika: Thanks, that's a good catch :) I'm so used to bash's [[ ]] semantics that mistakes like that creep in :/

@kualozach: `$UID` is a bash (and some other bourne shell implementation) variable that will break when you're running a strictly POSIX `sh`, which the installer script uses as its interpreter.